.github/docker-compose.yml

```
@@ -12,7 +12,7 @@ services:
       MARIADB_USER: magento
       MARIADB_PASSWORD: magento
   elastic:
-    image: elasticsearch:7.17.12
+    image: elasticsearch:7.17.13
```

Would it be better to ignore mention the minor releases?

```suggestion
    image: elasticsearch:7.17
```